tests/pytorch/nightly/hf-glue.libsonnet

```diff
@@ -167,7 +163,7 @@ local utils = import "templates/utils.libsonnet";
       ||| % {common: command_common, common_copy: command_copy_metrics}
     ),
     regressionTestConfig+: {
-      alert_for_failed_jobs: false,
```

Comment:
For my understanding, what does `alert_for_failed_jobs` do?

Reply:
if the GKE job ends in failure status, the metrics handler will normally send an alert. But if this setting is turned to `false`, then it won't trigger an alert. OOB metrics would still trigger alerts

https://github.com/GoogleCloudPlatform/ml-testing-accelerators/blob/master/metrics_handler/main.py#L529

We added this way back when you said that the tests were unstable. Note that the tests still show as failures in the dashboard, they just won't send email alerts